packages/commons/package.json

```diff
@@ -3,7 +3,11 @@
   "version": "1.0.0-alpha.3.1",
   "description": "OpenCRVS common modules and utils",
   "license": "MPL-2.0",
-  "main": "build/dist/index.js",
+  "main": "./build/dist/index.js",
+  "exports": {
```

Comment:
I've removed `@ef-carbon/tspm` because it didn't support this `exports` mapping, which was required to allow importing monitoring as `@opencrvs/commons/monitoring`. It was unmaintained anyways, so it was an excellent time to get rid of it.